lisa/util/excutableResult.py

```diff
@@ -0,0 +1,8 @@
+from typing import Optional
+
+
+class ExecutableResult:
+    def __init__(self, stdout: str, stderr: str, exitCode: Optional[int]) -> None:
```

Comment:
There has to a better alternative to this verbose pattern of argument -> property conversion...I'll do some research.

Reply:
Yes, a better way is necessary. I don't know if there is an easier way to define object level variables. Sometime I don't want to assign None to them in __init__.

Reply:
I found it! It's the Python [dataclasses](https://docs.python.org/3/library/dataclasses.html) module!